althea_types/src/interop.rs

```diff
@@ -744,3 +773,50 @@ pub struct HeartbeatMessage {
 pub struct ExitSystemTime {
     pub system_time: SystemTime,
 }
+#[cfg(test)]
+mod test {
+    pub fn data_serialize<S>(value: &Option<ContactType>, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let string_value = serde_json::to_string(&value).unwrap_or_default();
+        serializer.serialize_str(&string_value)
+    }
+
+    /// Deserializes a string as a ContactType
+    pub fn data_deserialize<'de, D>(deserializer: D) -> Result<Option<ContactType>, D::Error>
```

Comment:
you should be able to call the other functions? why duplicate? You just need a 'use' statement to import them at the top of the mod block.